packcheck.sh

```diff
@@ -1367,8 +1370,11 @@ then add them to .packcheck.ignore file at the root of the git repository."
   cd ${1}
   if test "$BUILD" = stack
   then
-    sdist_remove_project_file stack.yaml
-    $STACKCMD init
+    if test -z "$STACK_YAML"
+    then
+      sdist_remove_project_file stack.yaml
+      run_verbose_errexit $SDIST_STACKCMD init
+    fi
```

Comment:
Okay, we don't remove the `stack.yaml` if `STACK_YAML` is specified.

Reply:
We should do the same for `cabal.project` (`$CABAL_PROJECT`)?

Reply:
We do not need to do that for correctness, but for clarity we should do it. I will make the change.